pydrive2/test/README.rst

```diff
@@ -8,7 +8,9 @@ Run tests locally
     cd pydrive2/test/settings
     mkdir -p local && cp *.yaml local
 
-2. Setup a Google service account for your Google Cloud Project:
+2. Create **credentials** directory in the root directory of the repo.
```

Comment:
credentials is not gitignored

Reply:
but also see other comments - I would really love to avoid all these `mkdirs` in the root, even local dir